### ISSUE TITLE
SyntaxError: Unexpected token =

### ISSUE THREAD

**Issue by mandazi** [sahat/hackathon-starter#604]
First time using this and followed the guide and got this error upon trying to load the app for the first time:

```
$ nodemon app.js
[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node app.js`
/models/User.js:57
userSchema.methods.gravatar = function (size = 200) {
                                             ^

SyntaxError: Unexpected token =
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/controllers/user.js:5:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/app.js:33:24)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
    at startup (node.js:139:18)
    at node.js:968:3
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by sahat** [comment]
Duplicate of https://github.com/sahat/hackathon-starter/issues/597.

You will need to use Node.js 6.x as this project uses ES6 pretty extensively.

**Comment by mandazi** [comment]
Thanks!
